downloads/index.md

```diff
@@ -161,56 +161,140 @@ Checksums for this release are available in both, [MD5](https://julialang-s3.jul
-      </td>
-      <td colspan="3"> <a href="https://julialang-s3.julialang.org/bin/linux/x86/{{lts_release_short}}/julia-{{lts_release}}-linux-i686.tar.gz">32-bit</a>
-        (<a href="https://julialang-s3.julialang.org/bin/linux/x86/{{lts_release_short}}/julia-{{lts_release}}-linux-i686.tar.gz.asc">GPG</a>)
-      </td>
-    </tr>
-    <tr>
-      <th> Generic Linux Binaries for ARM <a href="/downloads/platform/#linux_and_freebsd">[help]</a></th>
-      <td colspan="3"> <a href="https://julialang-s3.julialang.org/bin/linux/aarch64/{{lts_release_short}}/julia-{{lts_release}}-linux-aarch64.tar.gz">64-bit (AArch64)</a>
-          (<a href="https://julialang-s3.julialang.org/bin/linux/aarch64/{{lts_release_short}}/julia-{{lts_release}}-linux-aarch64.tar.gz.asc">GPG</a>)
-      </td>
-      <td colspan="3"> <a href="https://julialang-s3.julialang.org/bin/linux/armv7l/{{lts_release_short}}/julia-{{lts_release}}-linux-armv7l.tar.gz">32-bit (ARMv7-a hard float)</a>
-        (<a href="https://julialang-s3.julialang.org/bin/linux/armv7l/{{lts_release_short}}/julia-{{lts_release}}-linux-armv7l.tar.gz.asc">GPG</a>)
-      </td>
-    </tr>
-    <tr>
-      <th> Generic FreeBSD Binaries for x86 <a href="/downloads/platform/#linux_and_freebsd">[help]</a></th>
-      <td colspan="3"> <a href="https://julialang-s3.julialang.org/bin/freebsd/x64/{{lts_release_short}}/julia-{{lts_release}}-freebsd-x86_64.tar.gz">64-bit</a>
-          (<a href="https://julialang-s3.julialang.org/bin/freebsd/x64/{{lts_release_short}}/julia-{{lts_release}}-freebsd-x86_64.tar.gz.asc">GPG</a>)
-      </td>
-      <td colspan="3"> </td>
-    </tr>
-    <tr>
-      <th> Source </th>
-      <td colspan="2"> <a href="https://github.com/JuliaLang/julia/releases/download/v{{lts_release}}/julia-{{lts_release}}.tar.gz">Tarball</a>
-            (<a href="https://github.com/JuliaLang/julia/releases/download/v{{lts_release}}/julia-{{lts_release}}.tar.gz.asc">GPG</a>)
-      </td>
-      <td colspan="2"> <a href="https://github.com/JuliaLang/julia/releases/download/v{{lts_release}}/julia-{{lts_release}}-full.tar.gz">Tarball with dependencies</a>
-        (<a href="https://github.com/JuliaLang/julia/releases/download/v{{lts_release}}/julia-{{lts_release}}-full.tar.gz.asc">GPG</a>)
-      </td>
-      <td colspan="2"> <a href="https://github.com/JuliaLang/julia/tree/v{{lts_release}}">GitHub</a> </td>
-    </tr>
-  </tbody>
-</table>
+.grid-container > div {
+  text-align: center;
+  padding: 20px 0 5 0;
+  font-size: 30px;
+}
+
+.downloads > p {
+  text-align: center;
+  margin: 6px;
+  font-size: 16px;
+}
+
+.grid-container h1 {
+  text-align: center;
+  margin: 16px;
+  font-size: 15px;
+}
+
+.downloads > h1 {
+  text-align: center;
+  margin: 16px;
+  font-size: 15px;
+}
+
+.downloads {
+  border: 2px solid #dddddd;
+  margin: 3px;
+  border-radius: 8px;
+}
+
+</style>
+
+<div class="grid-container">
+<!-- five columns -->
+	<div class="platform">
```

Comment:
Are tabs here intentional? Many HTML files in the website have this problem, of mixing tabs and spaces.

Reply:
This code was graciously donated to me by @cormullion (not pointing fingers just highlighting that other people may use spaces while I use tabs most of the time) but we can clean all these up after the content and design are set.

Reply:
Yeah, we should indent it to be clean.

Reply:
I plead guilty - my HTML formatting tool (BBEdit/TIdy) inserts tabs by default. 🤣